### PR TITLE
validation now created for signup and signin routes

### PR DESCRIPTION
**Pull request by fredie7**
pull request for ft-auth-request-validation

### CODE REVIEW

File: routers/auth.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import express from 'express';
 import authController from '../controllers/auth';
+import validate from '../validator';
```
Comment (jsamchineme) [inline routers/auth.js]: instead of **validate**, why not name this **validator**

**Review by jsamchineme — CHANGES_REQUESTED** [review]

File: routers/auth.js (hunk 1 above)
Comment (fredie7) [reply routers/auth.js]: I've made corrections..Thanks

File: controllers/auth.js [hunk 2]
```diff
@@ -35,7 +35,7 @@ const authController = {
   signin: (req, res) => {
     const existingUser = data.find((user) => user.email === req.body.email);
     if (!existingUser) {
-      return res.status(401).json({ error: 'user does not exist' });
+      return res.status(409).json({ error: 'user does not exist' });
```
Comment (jsamchineme) [inline controllers/auth.js]: The status code here should still be reverted to `401`, cos that's the appropriate value.This is not a conflict.

File: test/appTest.js [hunk 3]
```diff
@@ -1,11 +1,27 @@
...
-
-describe('App', () => {
-    it('app should run hello', ()=> {
-        assert.equal(app(), 'hello')
-    })
-    it('type should return a string', ()=> {
-        assert.typeOf(app(), 'string')
-    })
-})
+// const assert = require('chai').assert;
```
Comment (jsamchineme) [inline test/appTest.js]: delete the `test/appTest.js` file

**Review by jsamchineme — APPROVED** [review]

**Pull request merged**